File: android/src/main/kotlin/com/ecommpay/flutterplugin/models/PluginRecipientInfo.kt
package com.ecommpay.flutterplugin.models

import kotlinx.serialization.Serializable

@Serializable
internal data class PluginRecipientInfo(
    val walletOwner: String? = null,
    val walletId: String? = null,
    val country: String? = null,
    val pan: String? = null,
    val cardHolder: String? = null,
    val address: String? = null,
    val city: String? = null,
    val stateCode: String? = null,
)

File: ios/Classes/Models/PluginRecipientInfo.swift
//
//  PluginRecipientInfo.swift
//  ecmpplugin
//
//  Created by Alex on 19.04.2024.
//

import Foundation


internal struct PluginRecipientInfo: Decodable {
    let walletOwner: String?
    let walletId: String?
    let country: String?
    let pan: String?
    let cardHolder: String?
    let address: String?
    let city: String?
    let stateCode: String?
}
